tests/Makefile

```diff
@@ -73,6 +68,11 @@ rebuild: clean build
 
 framework:
 	make -C ../ framework TARGET=$(TARGET)
+	@echo -e ${GREEN}Copy Assets to [${BIN_DIR}/assets] ${NC}
```

Comment:
this is in the wrong place.

When your building the tests a requirement for running the tests is to copy the assets and any updates to the assets to run the tests

it has nothing to-do with building the framework.

we need to add a comment to state that it's doing.